sakai-evaluation-impl/src/java/org/sakaiproject/evaluation/dao/EvaluationDaoImpl.java

```diff
@@ -2023,6 +2033,37 @@ public int selectConsolidatedEmailRecipients(boolean useAvailableEmailSent, Date
     	return count;
 	}
 
+    /*
+     * (non-Javadoc)
+     * @see org.sakaiproject.evaluation.dao.EvaluationDao#getAllSiteIDsMatchingSectionTitle(java.lang.String)
+     */
+    public Set<String> getAllSiteIDsMatchingSectionTitle( String sectionTitleWithWildcards )
+    {
+        String sql = String.format( SQL_SELECT_SITE_IDS_MATCHING_SECTION_TITLE, sectionTitleWithWildcards );
+        sql = sql + SQL_SELECT_SITE_IDS_MATCHING_SECTION_TITLE_MODIFIER;
+
+        Session session = getSessionFactory().openSession();
+        SQLQuery query = session.createSQLQuery( sql );
+        Set<String> results = new HashSet<>( query.list() );
+        session.close();
+        return results;
+    }
```

Comment:
I'm particularly curious about if this is the best way to accomplish this query. Same goes with the one below.